devel/config/worker/osbuild-worker.toml

```diff
@@ -1,4 +1,4 @@
 [authentication]
-oauth_url = "http://fauxauth:8888/token?refresh_token=42"
+oauth_url = "http://fauxauth:8888/token?refresh_token=15885990"
```

Comment:
This isn't your personal org_id is it?

Reply:
I have no idea I can check after lunch. I think it's the default one that's set in Chloe's grafana PR though

Reply:
Ok checked. It's not my `org-id`

https://github.com/osbuild/osbuild-composer/pull/2756/files#diff-46ebf2e61d769f87c2459d6a997727fc5e16efa81f5a9b498406c8c3bf2ac4c6R2703-R2705